.zuul.yaml

```diff
@@ -80,6 +80,7 @@
               - cmd/cinder-csi-plugin/.*
               - manifests/cinder-csi-plugin/.* 
               - pkg/csi/cinder/.*
+              - pkg/cloudprovider/.*
```

Comment:
There are several basic structs that are inherited from those in pkg/cloudprovider, considering the changes we've made are mostly happening in service loadbalancer, I'd prefer we remove this dependency in CI for now.

If we really have concerns, the recommended approach would be removing the dependencies to pkg/cloudprovider.

Reply:
ok, so I think it takes time to split the code ... let me see what I can do